connaisseur/notary_api.py

```diff
@@ -23,6 +23,10 @@ def health_check(host: str):
     if not host:
         return False
 
+    # The ACR does not expose the notary health endpoint: https://github.com/Azure/acr/issues/475
+    if is_acr():
```

Comment:
what is the caveat of this like ACR notary is actually not healthy....?

Reply:
This basically translates into us not checking whether ACR is healthy, so Conny would show `Ready` even if the ACR were down, whereas for all other notaries Conny pods would become unhealthy once the connection to notary doesn't work anymore.
Do you think, I should make this more explicit here? Kinda thought it would be obvious from the code

Reply:
might make a comment in code or in the docs I guess...